RFCs/0023-TestSettings-Deprecation.md

```diff
@@ -0,0 +1,31 @@
+# 0023 TestSettings Deprecation
+
+## Summary
+
+This note outlines the proposed changes to deprecate support for testsettings in test runs.
+
+## Motivation
+
+Today there are two types of files to configure tests: *.testsettings and *.rusnettings. To simplify the experience, we are planning to deprecate support for *.testsettings.
```

Comment:
Wherever you say deprecate support, please qualify with 'for automated unit and functional testing scenarios'.